apps/emails/management/commands/email.py

```
@@ -0,0 +1,427 @@
+
+        return any(options.get(key) not in {None, False} for key in self.BRIDGE_EDITABLE_KEYS)
+
+    def _configure_inbox(self, options: dict[str, Any]) -> EmailInbox:
+        """Create or update an inbox profile from command options."""
+
+        inbox_id = options.get("inbox")
+        if inbox_id:
+            try:
+                inbox = EmailInbox.objects.get(pk=inbox_id)
+            except EmailInbox.DoesNotExist as exc:
+                raise CommandError(f"Inbox not found: {inbox_id}") from exc
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)